runelite-client/src/main/java/net/runelite/client/plugins/devtools/DevToolsOverlay.java

```diff
@@ -249,6 +255,34 @@ private void renderTileTooltip(Graphics2D graphics, Tile tile)
 		}
 	}
 
+	private void renderMovementInfo(Graphics2D graphics, Tile tile)
+	{
+		Polygon poly = Perspective.getCanvasTilePoly(client, tile.getLocalLocation());
+
+		if (poly != null && poly.contains(client.getMouseCanvasPosition().getX(), client.getMouseCanvasPosition().getY()))
```

Comment:
Use a guard clause instead of a large nested if statement.

eg.

```java
if (poly == null || !poly.contains(client.getMousePosition().getX(), client.getMousePosition().getY())
{
	return;
}

// continue method code here
```